app/views/products/show.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="row1">
+  <div class="column1">
+    <div class="product-img-div">
```

Comment:
Thanks a lot Yassine, you can also implement less HTML in the future (also because we are using grid and flexboxes, this is really similar to bootstrap grid in truth 😂).